spec/confirmation_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'confirmation'
+
+describe Confirmation do
+  subject(:confirmation) { described_class.new }
+  let(:payment) { :payment }
+
+  it 'raises an error if the money does not match the total' do
+    expect { subject.checkout(payment) }.to raise_error(RuntimeError,'something went wrong, try again')
```

Comment:
Clear message when raising an error, is `RuntimeError` needed in the parameter?

Reply:
This syntax can be used to specify the error, so I tried it. 
Example:  expect { raise "oops" }.to raise_error(RuntimeError, "oops")